components/slices/ImageBanner.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <section class="image-banner" :class="{
+    'align-right': fields.banner_alignment === 'right',
+    'align-left': fields.banner_alignment === 'left',
+    'align-center': fields.banner_alignment === 'center',
+    'theme-light': fields.banner_theme === 'light',
+    'theme-dark': fields.banner_theme === 'dark'
+  }">
+    <banner-content
+      :title="fields.banner_title[0].text"
+      :content="fields.banner_content">
+    </banner-content>
+    <picture-component
+      :desktopUrl="fields.banner_image.url"
+      :mobileUrl="fields.banner_image.Mobile.url"
+      :imageAlt="fields.banner_image.alt"
+    >
+    </picture-component>
+  </section>
+</template>
+
+<script>
+
+  import BannerContent from "../banners/BannerContent";
+  import PictureComponent from "../utility/PictureComponent";
+
+  export default {
+    components: {
+      PictureComponent,
+      BannerContent},
+    name: "image-banner",
+    data() {
+      return {
+       // title: "color-quote"
+      }
+    },
+    props: {
+      fields: Object
+    }
+  }
+</script>
+
+<style lang="scss">
```

Comment:
Not scoped so child content can inherit styles.

Reply:
I think this should be scoped, and then don't style dom elements that live in the child components...but add classes to your child components and style those directly